lib/scope.js

```diff
@@ -794,35 +813,55 @@ module.exports = {
   },  // Ends scope.load()
 
   getLoadData: async function ( scope ) {
-    /* Return one of two elements that could load on the page - #daMainQuestion
-    * or #da-retry. If there's a system error on the page, return its main text.
+    /* Return elements that could load on the page, like #daMainQuestion
+    * and #da-retry. If there's a system error on the page, return its main text.
     * 
     * If server isn't loaded yet, it should get the browser timeout error,
-    * which won't match either of those and this will timeout and, I assume, error.*/
+    * which won't match those selectors and this will timeout and error. */
     let result = { error: ``, elem: null };
 
+    // Wait for at least one of these to appear
     let winner = await Promise.race([
       // Signature page
       scope.page.waitForSelector( `#dasigpage`, { timeout: scope.timeout }),
       // Any other valid page
       scope.page.waitForSelector( `#daMainQuestion`, { timeout: scope.timeout }),
       scope.page.waitForSelector( `#da-retry`, { timeout: scope.timeout }),
-    ]);
+    ]).catch( function ( error ) {
```

Comment:
probably out of scope for this PR, but for my own curiosity:

Can we tell when the page is completely finished loading? Would mean that we wouldn't have to wait for the timeout when everything is done and sitting there.

Reply:
We already do that by checking for any of those elements to be present. As soon as one is there, the race stops and the code moves on. The timeout is just in case none of them are found. We don't want to wait for them for forever. Of course, we may be missing some kind of indicator of a page having loaded, but I'm not sure how to track that down.

Reply:
> The timeout is just in case none of them are found. We don't want to wait for them for forever. 

That was my question, if none of them appear, we just wait for the timeout, which is a little silly IMO, especially for tests with longer timeouts.

This stackoverflow has some code that we should consider using: https://stackoverflow.com/questions/52497252/puppeteer-wait-until-page-is-completely-loaded.

Reply:
Always good to have a new eye on things! When I looked, I only remember finding stuff about `networidle` being unreliable. This looks like an interesting solution. We should back it up with the timeout as well, though, just in case it keeps changing for some reason. See [the comment below the answer](https://stackoverflow.com/questions/52497252/puppeteer-wait-until-page-is-completely-loaded#comment118988927_61304202) for an example.